cumulusci.yml

```
@@ -54,6 +54,23 @@ tasks:
             path: unpackaged/config/spe
             namespace_inject: $project_config.project__package__namespace
 
+    deploy_language_translation_workbench_settings:
+        description: Deploys configuration for translation wokbench enablement. Enables end user languages, platform languages, and translation workbench.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            namespace_inject: $project_config.project__package__namespace
+            path: unpackaged/config/qa/language/settings
+            unmanaged: True
+
+    deploy_translation_settings:
+        description: Deploys configuration for translation workbench language supported language enablement. Assigns English (British),Spanish, Spanish (Mexico),French as active translantion workbench supported languages. Must have translation workbench and platform languages enabled for this task to work.
```

I agree with Sarah's suggestion:
```suggestion
        description: Assigns languages and marks them as active.
```